src/routes/dashboard/helpers/contributors.ts

```diff
@@ -0,0 +1,47 @@
+import type { Contributor, GitHubUser } from '../types'
+
+export function getGitHubUsername(gitHubStaff: GitHubUser[], userId: string) {
+  const user =
+    gitHubStaff[gitHubStaff.findIndex((user) => user.id === Number(userId))]
+  if (user?.login) return user.login
+  return ''
+}
+
+export async function getTopContributors(
+  contributors: Contributor[],
+  gitHubStaff: GitHubUser[],
+  n: number
+) {
+  const filtered = contributors
+    .sort((prev, next) => next.answers.length - prev.answers.length)
+    .slice(0, n)
+  const result = []
+  for (const contributor of filtered) {
+    let ghUsername = ''
+    let name = ''
+    if (contributor.githubId) {
+      ghUsername = getGitHubUsername(gitHubStaff, contributor.githubId)
+    }
+    if (ghUsername) {
+      name = await (
+        await fetch(`${import.meta.env.VITE_HOST}/api/github/${ghUsername}`, {
+          method: 'GET',
```

Comment:
fwiw, `GET` is the default method for `fetch` and can be omitted https://developer.mozilla.org/en-US/docs/Web/API/Request/Request#syntax